src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFilesAccordion.tsx

```diff
@@ -0,0 +1,115 @@
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import Details from '@common/components/Details';
+import { Release } from '@common/services/publicationService';
+import { FileInfo } from '@common/services/types/file';
+import React, { ReactNode } from 'react';
+import styles from './ReleaseDataAndFilesAccordion.module.scss';
+
+interface Props {
+  release: Release;
+  renderCreateTablesButton?: ReactNode;
+  renderDownloadLink: (file: FileInfo) => ReactNode;
+  renderMetaGuidanceLink: ReactNode;
+  renderPreReleaseAccessLink?: ReactNode;
+  onSectionOpen?: (accordionSection: { id: string; title: string }) => void;
+}
+const ReleaseDataAndFilesAccordion = ({
+  release,
+  renderCreateTablesButton,
+  renderDownloadLink,
+  renderMetaGuidanceLink,
+  renderPreReleaseAccessLink,
+  onSectionOpen,
+}: Props) => {
+  const dataFiles = release.downloadFiles.filter(
+    file => file.name === 'All files' || file.type !== 'Ancillary',
+  );
+
+  const ancillaryFiles = release.downloadFiles.filter(
+    file => file.type === 'Ancillary' && file.name !== 'All files',
+  );
+
+  return (
+    <div className={styles.container}>
+      <Accordion
+        id="dataDownloads"
+        showOpenAll={false}
+        onSectionOpen={accordionSection => {
+          if (onSectionOpen) {
+            onSectionOpen(accordionSection);
+          }
+        }}
+      >
+        <AccordionSection heading="Explore data and files">
+          {dataFiles.length > 0 && (
+            <>
+              <p>
+                All data used to create this release is published as open data
+                and is available for download.
+              </p>
+              <p>
+                You can create your own tables from this data using our table
+                tool, or view featured tables that we have built for you.
+              </p>
+
+              <ul className="govuk-list">
+                {dataFiles.map(file => {
+                  return <li key={file.id}>{renderDownloadLink(file)}</li>;
+                })}
+              </ul>
+            </>
+          )}
+          {renderCreateTablesButton && (
+            <div className="dfe-flex dfe-justify-content--space-between dfe-align-items--center">
+              <div>
+                <h3>Create your own tables</h3>
+                <p>
+                  Explore our range of data and build your own tables from it.
+                </p>
+              </div>
+              <p className="govuk-!-width-one-quarter dfe-flex-shrink--0">
+                {renderCreateTablesButton}
+              </p>
```

Comment:
A few little things here:

- This is probably better as a `div` than a `p`
- Don't think we need the `dfe-flex-shrink--0`